mp_api/client/routes/__init__.py

```diff
@@ -30,4 +29,20 @@
 from ._user_settings import UserSettingsRester
 from ._general_store import GeneralStoreRester
 from .bonds import BondsRester
-from .alloys import AlloysRester
+from .robocrys import RobocrysRester
+
+try:
+    from .alloys import AlloysRester
+except ImportError:
+    import warnings
+    warnings.warn("Alloy addon package not installed. "
+                  "To query alloy data install the pymatgen-analysis-alloys package.")
```

Comment:
Looks like this warning will appear every time the client is initialized without `pymatgen-analysis-alloys` being installed even if the user isn't interested in alloy data. Maybe this could be moved to the part of the code that tries to retrieve alloy data?

Reply:
Yeah, I debated adding a warning on instantiation. I may just remove this...